rust_src/wrapper.h

```diff
@@ -61,3 +61,4 @@
 #endif
 #include "window.h"
 #include "xgselect.h"
+#include "xwidget.h"
```

Comment:
Why is this here?

Reply:
If it's removed `kill_buffer_xwidgets` is not added to `remacs_sys`